mgmt/obj/ftn.py

```diff
@@ -0,0 +1,116 @@
+
+import logging
+from kubernetes.client.rest import ApiException
+
+logger = logging.getLogger()
+
+class Ftn(object):
+	def __init__(self, obj_api, opr_store, spec=None):
+		self.obj_api = obj_api
+		self.store = opr_store
+		self.droplet_obj = None
+		self.vpc = ""
+		self.vni = ""
+		self.nip = ""
+		self.prefix = ""
+		self.net = ""

```

Comment:
Why do you need to have these here? An FTN object shall carry the following properties (these should match your CRD definition.. No ? ):
```
FTN Node:
    - Name = "T1"
    - DropletName = "D1"
    - IP = "172.0.0.1"
    - Mac = "a:b:c:d:e:1"
    - Chains = ["C1"] // At least one chain (maximum two)
    - NextReplica = ["T2"]
```